### ISSUE TITLE
Versions differ: 2014-12-23 (slime) vs. 2013-06-26 (swank). Continue? (y or n)

### ISSUE THREAD

**Issue by staykov** [slime/slime#222]
Hello,
Ever since I updated to the new slime 2.12 in order to reslove some SBCL compiling issues in Emacs, I now get this message every time I run slime:
"Versions differ: 2014-12-23 (slime) vs. 2013-06-26 (swank). Continue? (y or n)"

How do I fix this? Thanks.

OS: Ubuntu 14.04 LTS 64bit

**Comment by luismbo** [comment]
So, SLIME is picking up a very old SWANK, somehow...

What value does `slime-path` have on the Emacs side? What about `swank-loader:*source-directory*` on the Lisp side?

**Comment by staykov** [comment]
1) 
 ;; Set up the Common Lisp environment
 (add-to-list 'load-path "/usr/share/common-lisp/source/slime-2.12")
 (require 'slime-autoloads)
 (setq inferior-lisp-program "/usr/bin/sbcl")
 (require 'slime)
 (slime-setup)

2)
I'm not sure where to check the swank-loader dir.

**Comment by luismbo** [comment]
Let me try again. :-) Please tell me two things:

1. What's the value of `slime-path` on the Emacs side. You can determine this by opening the elisp console with `M-x ielm` then evaluate `slime-path`  there. 

2. What's the value of `swank-loader:*source-directory*` on the Lisp side via the console you get after starting SLIME with `M-x slime`.

**Comment by staykov** [comment]
1) ELISP> slime-path
"/usr/share/common-lisp/source/slime-2.12/"

2) (progn (load "/usr/share/common-lisp/source/slime/swank-loader.lisp" :verbose t)

Thank you figured it out with your help, apparently I kept two different versions of slime. I deleted the older one and renamed the new folder to just "slime". Now it works like a charm.

**Comment by bigos** [comment]
Luismbo has suggested checking slime and swank paths. In my case both pointed to 

> "/home/jacek/quicklisp/dists/quicklisp/software/slime-v2.23/"

However I had also another version of slime installed from MELPA. Also in my Emacs config I was calling to various slime configuration functions BEFORE requesting quicklisp version of slime.
Moving requesting slime from quicklisp to the top seems to have solved the issue that was bothering me for months. This commit has fixed the problem in my Emacs config. I hope it helps someone.

> https://github.com/bigos/prelude/commit/756dc52603a38d8eaf0f2f8fa8494a6eeb091b34
